components/ChainTree/BlockPreview.tsx

```diff
@@ -32,7 +33,9 @@ const PREVIEW_BLOCKS = [
   },
   {
     label: 'Graffiti',
-    value: safeProp('graffiti'),
```

Comment:
Why do we have to render the `Box` here now?

Reply:
Removed